### PR TITLE
Add missing classes

### PR DESCRIPTION
**Pull request by ryantmer**
Just some classes that we're using the notifications pipeline that I noticed were missing.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -185,6 +185,7 @@ export const Classes = {
 		assignmentActivity: 'assignment-activity',
 		complete: 'complete',
 		completion: 'completion',
+		completionTracked: 'completion-tracked',
```
Comment (ryantmer) [inline index.js]: @hekard2l Does `completion-tracked` make sense as being part of the `activities` namespace? `completion` is used only in the activities domain, but seems like `completion-tracked` is part of the `completionTracking` project.

File: index.js [hunk 2]
```diff
@@ -223,11 +224,15 @@ export const Classes = {
 		userAward: 'user-award',
 		userAwards: 'user-awards'
 	},
+	checklists: {
+		checklistItem: 'checklist-item'
```
Comment (ryantmer) [inline index.js]: @ryantmer You should have added this when you made the checklists HM API.

File: index.js [hunk 3]
```diff
@@ -279,6 +284,9 @@ export const Classes = {
 		quiz: 'quiz',
 		description: 'description'
 	},
+	surveys: {
+		survey: 'survey'
```
Comment (ryantmer) [inline index.js]: @ryantmer You forgot this one too.

File: index.js (hunk 2 above)
Comment (ryantmer) [reply index.js]: "Damn, present-day me, you're so much smarter than I am" - me from a year ago

File: index.js (hunk 3 above)
Comment (ryantmer) [reply index.js]: Wow I really dropped the ball on this.

File: index.js (hunk 1 above)
Comment (hekard2l) [reply index.js]: Hmm... `completion-tracked` is a class for `Organizations`. I'd personally put it on `line 281`

Comment (ryantmer) [reply index.js]: Done! Wasn't really sure where it fit logically-speaking, so thank you! 💯

**Pull request merged**